src/views/getProfile.js

```diff
@@ -12,8 +12,8 @@ export class Profile extends Component {
   }
 
   render() {
-    const { fetched } = this.props;
-    if (!fetched) {
+    const { isFetching } = this.props;
+    if (!isFetching) {
```

Comment:
Why should I have a loader while there is success or failure? For example, when getting a profile, when a request is sent, isfetching is true while in case of failure or success, then it is false. 
This code means that loader is shown when there is success or failure and not while loading. Please check on this and test it.